main/ertp/guide/create-fungible-erights.md

```diff
@@ -0,0 +1,92 @@
+# Creating a fungible eright with ERTP
+
+## Definition of "fungible"
+
+A **fungible** asset means that different elements of this asset are interchangeable
+Currencies are interchangeable: if you have 10 one-dollar notes laying around and decide
+to take 5 of them, you do not care which 5 you will take. They're all "the same" as far 
+as counting value goes.
+
+This would not be the case for 10 house ownership rights each designating a different house
+House ownership right are not fungible, they are **specific**
```

Comment:
I think this should be "ownership rights". We can add some periods in this section too